lesson_28/main_page.py

```diff
@@ -0,0 +1,86 @@
+from lesson_28.abstract_page import AbstractPage
+from selenium.webdriver.common.by import By
+from selenium.webdriver.ie.webdriver import WebDriver
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+
+class MainPage(AbstractPage):
+
+    def __init__(self, driver: WebDriver):
+        super().__init__(driver)
+
+    @property
+    def sign_in_button(self):
+        wait = WebDriverWait(self._driver, 30)
+        return wait.until(
```

Comment:
This method should be created inside at least AbstractElement class in case to avoid code duplication as you did in your pages classes.